src/TestBench.js

```diff
@@ -212,7 +212,8 @@ export default class TestBench extends React.Component {
           value={t.id}
           checked={current === t.id}
           onChange={this.onReconTypeChange}>
-        {t.name}<br /><span className="reconTypeId">{t.id}</span>
+        {t.name}<br />
+        {Array.isArray(t.broader) && t.broader.length && <span className="reconTypeId">{t.broader.map(e => e.id).join(', ')} &gt; </span>}<span className="reconTypeId">{t.id}</span>
```

Comment:
In general, in a user interface I would rather display the `name` of the type rather than the `id`, but this is a test bench, so arguably displaying ids makes sense too!